### ISSUE TITLE
Expand Index details

### ISSUE THREAD

**Issue by marcingminski** [marcingminski/sqlwatch#143]
From Slack channel: 

> I am thinking not sure but I see two pages for indexes. Great info on their usage and missing index impact. But for dev and dba's wouldn't that be nice to also show may be the details of indexes like on which columns are they created and which are leading key columns etc .. This can help understand tables as well

**Comment by ClaudioESSilva** [comment]
When I do analysis of indexes that is something I have in mind. Key Columns + Include Columns also the compression type.
That said, having this extra info can be helpful.

**Comment by FrenchDataWizard** [comment]
This would indeed be useful. However, maybe split it into 2 dashboards for maintenance and performance considerations.
On more DBA oriented with monitoring and maintenance in mind.
The other for DEVs, with debug of index and performance in mind.

**Comment by marcingminski** [comment]
Yeah, I'm splitting the dashboard up for performance reasons anyway. Pulling data takes ~8 seconds but parsing and building pbi schema could take ~2 minutes with every refresh. PowerBI is great with data but not with many tables.
